include/solana/pushcompact.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
```

Comment:
would call this module CompactU16 and then just define all methods with the same name:

void encode(std::vector,..)
void encode(uint16_t,..)